stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-13.8
+resolver: lts-13.24
```

Comment:
Wow! Careful with this :smile: ... Now we need to reset all the caches in CI because they'll likely have doubled in size.

Reply:
> Wow! Careful with this smile ... Now we need to reset all the caches in CI because they'll likely have doubled in size.

Good point. The first time Travis built this, the build time nearly exceeded the 50 minutes limit. Do we have a defined process for changing the resolver?

Reply:
Not necessarily, in the sense that, this _should_ work just fine. But we have to make sure to remove the cache from `master` before merging. Otherwise, we end up with a cache twice as big on `master` because it'll carry two LTS. Although, this will be needed for a short while until everyone gets existing PRs up-to-date.

It's just that, when I see "Make DB QSM tests fail ...", I don't expect an LTS bump :sweat_smile: ... It's good to advertise this kind of change on our Slack channel because it impacts everyone.

Reply:
> But we have to make sure to remove the cache from master before merging.

Just to confirm: Can this be done by navigating to https://travis-ci.org/input-output-hk/cardano-wallet/caches and then hitting the garbage can icon?

> It's just that, when I see "Make DB QSM tests fail ...", I don't expect an LTS bump sweat_smile ... It's good to advertise this kind of change on our Slack channel because it impacts everyone.

Good point.

Perhaps this is something we can add to our list of things to take care of when writing a PR / doing a review.

Reply:
> Just to confirm: Can this be done by navigating to https://travis-ci.org/input-output-hk/cardano-wallet/caches and then hitting the garbage can icon?

Indeed.

> Perhaps this is something we can add to our list of things to take care of when writing a PR / doing a review.

:+1: